dissect/target/helpers/configutil.py

```diff
@@ -228,6 +228,50 @@ def parse_file(self, fh: TextIO) -> None:
         self.parsed_data = information_dict
 
 
+class OpenVPN(Default):
```

Comment:
This is an application specific parser, I'm not sure it belongs in this file. Is there no way to achieve something similar with this being in the `openvpn.py` file?